src/main/java/wolox/training/models/Book.java

```diff
@@ -51,4 +57,8 @@
     @NotNull
     private String isbn;
 
+    @ManyToOne
```

Comment:
So if there is a many-to-one relationship between Book and User that means that a User can have multiple books, but each Book only can have a relation with only one user. That does not seem ok, check the specification of the card that ask you that a Book can be in the collection of more than one User.